overlord/devicestate/firstboot.go

```diff
@@ -386,27 +379,23 @@ func (m *DeviceManager) importAssertionsFromSeed(isCoreBoot bool) (seed.Seed, er
 // processAutoImportAssertions attempts to load the auto import assertions
 // and create all knows system users, if and only if the model grade is dangerous.
 // Processing of the auto-import assertion is opportunistic and should not fail
```

Comment:
now that this returns an error the comment is a bit confusing, maybe it's better to remove it and have it before the call to ensure

Reply:
fixed